pkg/cli/aws/aws.go

```diff
@@ -51,6 +51,25 @@ func DefaultCredentialOptions() *CredentialOptions {
 	}
 }
 
+// DefaultCredentialOptionsWithLog uses the default role and profile
+// for accessing AWS.
+func DefaultCredentialOptionsWithLog(log logrus.FieldLogger) *CredentialOptions {
+	b, err := box.LoadBox()
+	if err != nil {
+		return nil
+	}
```

Comment:
suggestion: if `log` is required, you can print an error message here.
```suggestion
	if err != nil {
		log.Errorf("Could not load box config: %w", err)
		return nil
	}
```